apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -740,7 +740,7 @@ function initializeBlocklyWrapper(blocklyInstance: GoogleBlocklyInstance) {
         connectionChecker: CdoConnectionChecker,
       },
       renderer: optOptionsExtended.renderer || Renderers.DEFAULT,
-      comments: false,
+      comments: !!optOptionsExtended.comments,
```

Comment:
Would this get taken care of by the spread on `optOptionsExtended` above? Or do we want to make sure we explicitly set this to `false` if there's no `comments` value?

Reply:
If there's no value the `!!` will convert undefined to false. I confirmed this with Sprite Lab: `options.comments` is false and the feature is not available. The way it's written explicitly requires a truthy value to be passed into the inject options.

Reply:
Ah interesting, asking because I see that the `comment` field is [optional](https://developers.google.com/blockly/reference/js/blockly.blocklyoptions_interface.comments_propertysignature.md) on the options interface - but sounds like we need to explicitly pass in a truthy value?

Reply:
We need to explicitly pass in a truth value if we want the feature turned on. See the change in MusicBlocklyWorkspace for where we can start doing this.

Reply:
Yep, sorry to clarify, I just mean that if a caller is already passing in `comment: true`, that value will automatically get added to the `options` object since it's constructed by extending `optOptionsExtended`. So wondering if that's sufficient, or if the reason to keep this line here is that we explicitly want `comment` to be `false` if no `comment` value is provided in the `optOptions` argument to `inject`

Reply:
> we explicitly want `comment` to be `false` if no comment value is provided in the `optOptions` argument to `inject`

This! I was seeing that without this, we seemed to always be enabling comments, even in old labs that didn't specify. In fact, it seems that if `options.comments` is undefined, Blockly decides based on whether or not categories are used in the toolbox:

https://github.com/google/blockly/blob/5fd337bb4d12b68766cae8094893ed0131285666/core/options.ts#L103-L105

Reply:
Ah gotcha, interesting! Thanks for clarifying 🙂